src/main/cpp/subsystems/DriveSubsystem.cpp

```diff
@@ -170,8 +171,13 @@ void DriveSubsystem::logDrivebase() {
                             m_rearLeft.GetState(), m_rearRight.GetState()};
   std::span<frc::SwerveModuleState, 4> states(states_vec.begin(),
                                               states_vec.end());
+  std::vector desired_vec = {m_frontLeft.GetDesiredState(), m_frontRight.GetDesiredState(),
```

Comment:
Please don't take advantage of goofy C++20; please use a template so types are obvious.

Reply:
@WowMuchDoge You should take the lead on implementing this